wgpu-core/src/validation.rs

```diff
@@ -1047,6 +1057,22 @@ impl Interface {
             }
         }
 
+        // check workgroup size limits
+        if shader_stage == naga::ShaderStage::Compute {
+            let max_workgroup_size_limits = [
+                self.limits.max_compute_workgroup_size_x,
+                self.limits.max_compute_workgroup_size_y,
+                self.limits.max_compute_workgroup_size_z,
+            ];
+
+            if entry_point.workgroup_size > max_workgroup_size_limits {
```

Comment:
I think this is a wrong way to check. You'd get `[1, 10000, 1] > [16,16,16]` failing.

Reply:
I changed to the _standard_ way of checking. I don't know why I thougth it could be that way lol